src/components/StoriesNavigation/TabPanel.js

```diff
@@ -0,0 +1,33 @@
+import { Grid } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+
+function TabPanel({ children, value, index, ...other }) {
+  return (
+    <Grid
+      container
+      direction="column"
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && <div>{children}</div>}
```

Comment:
We have already set `hidden = {value !== index}`, There's no need of checking again here